From 1656d858b19d6690cfb3894daff8423d9edcf6e1 Mon Sep 17 00:00:00 2001
From: Sandy Huang <hjc@rock-chips.com>
Date: Tue, 14 Jul 2020 18:02:39 +0800
Subject: [PATCH] Revert "drm/modes: add dclk invert config parse"

This reverts commit 4492caa259800fb30c725f4aa72ef00c37f82f71.

remove rockchip private flag.
use DRM_BUS_FLAG_PIXDATA_NEGEDGE to instead of DRM_MODE_FLAG_PPIXDATA.

Change-Id: I24ce988a84a4a5793f7eb3e26e51293988d854d0
Signed-off-by: Sandy Huang <hjc@rock-chips.com>
---
 drivers/gpu/drm/drm_modes.c | 4 ----
 include/uapi/drm/drm_mode.h | 5 +----
 2 files changed, 1 insertion(+), 8 deletions(-)

diff --git a/drivers/gpu/drm/drm_modes.c b/drivers/gpu/drm/drm_modes.c
index 7325d233a695..71d6adfd9e00 100644
--- a/drivers/gpu/drm/drm_modes.c
+++ b/drivers/gpu/drm/drm_modes.c
@@ -612,8 +612,6 @@ void drm_display_mode_from_videomode(const struct videomode *vm,
 		dmode->flags |= DRM_MODE_FLAG_DBLSCAN;
 	if (vm->flags & DISPLAY_FLAGS_DOUBLECLK)
 		dmode->flags |= DRM_MODE_FLAG_DBLCLK;
-	if (vm->flags & DISPLAY_FLAGS_PIXDATA_POSEDGE)
-		dmode->flags |= DRM_MODE_FLAG_PPIXDATA;
 	drm_mode_set_name(dmode);
 }
 EXPORT_SYMBOL_GPL(drm_display_mode_from_videomode);
@@ -655,8 +653,6 @@ void drm_display_mode_to_videomode(const struct drm_display_mode *dmode,
 		vm->flags |= DISPLAY_FLAGS_DOUBLESCAN;
 	if (dmode->flags & DRM_MODE_FLAG_DBLCLK)
 		vm->flags |= DISPLAY_FLAGS_DOUBLECLK;
-	if (dmode->flags & DRM_MODE_FLAG_PPIXDATA)
-		vm->flags |= DISPLAY_FLAGS_PIXDATA_POSEDGE;
 }
 EXPORT_SYMBOL_GPL(drm_display_mode_to_videomode);
 
diff --git a/include/uapi/drm/drm_mode.h b/include/uapi/drm/drm_mode.h
index 91c30e12ae50..3b8571193043 100644
--- a/include/uapi/drm/drm_mode.h
+++ b/include/uapi/drm/drm_mode.h
@@ -116,8 +116,6 @@ extern "C" {
 #define  DRM_MODE_FLAG_PIC_AR_256_135 \
 			(DRM_MODE_PICTURE_ASPECT_256_135<<19)
 
-#define DRM_MODE_FLAG_PPIXDATA			(1<<31)
-
 #define  DRM_MODE_FLAG_ALL	(DRM_MODE_FLAG_PHSYNC |		\
 				 DRM_MODE_FLAG_NHSYNC |		\
 				 DRM_MODE_FLAG_PVSYNC |		\
@@ -130,8 +128,7 @@ extern "C" {
 				 DRM_MODE_FLAG_HSKEW |		\
 				 DRM_MODE_FLAG_DBLCLK |		\
 				 DRM_MODE_FLAG_CLKDIV2 |	\
-				 DRM_MODE_FLAG_3D_MASK |	\
-				 DRM_MODE_FLAG_PPIXDATA)
+				 DRM_MODE_FLAG_3D_MASK)
 
 /* DPMS flags */
 /* bit compatible with the xorg definitions. */
-- 
2.35.3

